wrappers/csharp/src/Processing.cs

```diff
@@ -102,4 +102,25 @@ public FrameSet Process(FrameSet original)
 
         FrameQueue queue;
     }
+
+    public class PointCloud : ProcessingBlock
+    {
+        FrameQueue queue;
+
+        public PointCloud()
+        {
+            object error;
+            m_instance = new HandleRef(this, NativeMethods.rs2_create_pointcloud(out error));
+            queue = new FrameQueue();
+            NativeMethods.rs2_start_processing_queue(m_instance.Handle, queue.m_instance.Handle, out error);
+        }
+
+        public Points Points(Frame original)
```

Comment:
I think we should stick to the name as it appears on other APIs - i.e `Calculate(Frame original)`

Reply:
Surely that's right. I will fix the method name.